src/NServiceBus.Core/Audit/RouteToAudit.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.Audit
+{
+    using System.Collections.Generic;
+    using NServiceBus.Performance.TimeToBeReceived;
+    using NServiceBus.Routing;
+    using Pipeline;
+    using Transport;
+
+    /// <summary>
+    /// Default action that routes the audit message to the configured audit queue.
+    /// </summary>
+    public class RouteToAudit : AuditAction
+    {
+        /// <summary>
```

Comment:
Should we make the ctor `protected internal` here to make sure we create it internally or you have to inherit from it?

Reply:
done